cmd/zoekt-sourcegraph-indexserver/main_test.go

```diff
@@ -17,14 +17,13 @@ import (
 
 	sglog "github.com/sourcegraph/log"
 	"github.com/sourcegraph/log/logtest"
-
+	proto "github.com/sourcegraph/zoekt/cmd/zoekt-sourcegraph-indexserver/protos/sourcegraph/zoekt/configuration/v1"
```

Comment:
not sure why your editor is grouping this here. Looks like where it used to live makes more sense? Same thing happens in other places.